easybuild/easyblocks/s/superlu.py

```diff
@@ -123,16 +123,19 @@ def install_step(self):
         """
         super(EB_SuperLU, self).install_step()
 
-        expected_libpath = os.path.join(self.installdir, "lib", "libsuperlu.%s" % self.lib_ext)
-        actual_libpath   = os.path.join(self.installdir, "lib", "libsuperlu_%s.%s" % (self.cfg['version'], self.lib_ext))
+        libbit = "lib"
+        if not os.path.exists(os.path.join(self.installdir, libbit)):
+                libbit = "lib64"
```

Comment:
@hajgato please fix indent here (4 spaces)

Also, what if the `lib64` dir doesn't exist either? Don't we want a clean error message then?